### ISSUE TITLE
ask_open_file causes Gtk-WARNING

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#300]
A simple `ask_open_file` on Windows triggers the following error:

```
(cshoes.exe:12316): Gtk-WARNING **: Could not find the icon 'drive-optical-symbolic'. The 'hicolor' theme
was not found either, perhaps you need to install it.
You can get a copy from:
        http://icon-theme.freedesktop.org/releases
```

**Comment by ccoupe** [comment]
This one was typo in the cross compile build script (as you'll see in the commit). I have no idea how long its been there.  I've updated the exe at the download site with a fixed version, still 3.3.2

Thanks for reporting this.

**Comment by BackOrder** [comment]
You're welcome. We started using the issue tracker two years ago and 160 issues closed later, and counting, here we are. Shoes are shining.
